apps/web/app/modules/Statistics/components/RatesChart.tsx

```diff
@@ -0,0 +1,122 @@
+import { Bar, BarChart, CartesianGrid, XAxis, YAxis } from "recharts";
+
+import { ChartContainer, ChartTooltip, ChartTooltipContent } from "~/components/ui/chart";
+import { Skeleton } from "~/components/ui/skeleton";
+import { ChartLegendBadge } from "~/modules/Statistics/components/ChartLegendBadge";
+
+import type { ChartConfig } from "~/components/ui/chart";
+
+const chartConfig = {
+  completed: {
+    label: "Completed",
+    color: "var(--primary-700)",
+  },
+  started: {
+    label: "Started",
+    color: "var(--primary-300)",
+  },
+} satisfies ChartConfig;
+
+interface ChartData {
+  month: string;
+  completed: number;
+  started: number;
+}
+
+type RatesChartProps = {
+  isLoading?: boolean;
+  resourceName: string;
+  chartData: ChartData[];
+};
+
+export const RatesChart = ({ isLoading = false, resourceName, chartData }: RatesChartProps) => {
+  const dataMax = Math.max(...chartData.map((d) => d.started));
```

Comment:
Change the name to something more descriptive. d -> data. 

Also i think it is a good idea to use useMemo in these 4 variables